src/Pulp/Init.purs

```diff
@@ -153,42 +153,50 @@ init initStyle effOrEffect force out = do
       launchBower ["install", "--save-dev", p.psciSupport ]
 
     install PscPackage UseEff _ = do
+      failIfUsingPscPackageAndEsModules
+
       launchPscPackage ["init"]
       launchPscPackage ["install", "eff"]
       launchPscPackage ["install", "console"]
       launchPscPackage ["install", "psci-support"]
 
     install PscPackage UseEffect _ = do
+      failIfUsingPscPackageAndEsModules
+
       launchPscPackage ["init"]
       launchPscPackage ["install", "effect"]
       launchPscPackage ["install", "console"]
       launchPscPackage ["install", "psci-support"]
 
+    failIfUsingPscPackageAndEsModules = do
+      failIfUsingEsModulesPsVersion out $ Just
+        "'psc-package' not yet supported on a `purs` version that compiles to ES modules."
```

Comment:
```suggestion
        "'psc-package' is not supported with `purs` compilers that emit ES modules."
```

Reply:
As far as I understand, `psc-package` can be supported. So, I think we should leave this as is until someone implements the work to do that. I don't personally use `psc-package`, but if someone else does, then they can implement that code if they're using `pulp`.